### DISCUSSION TITLE
[Experimental] rolling-update Updater

### DISCUSSION THREAD

**Discussion by sr229** [Monika-After-Story/MonikaModDev#7832] [general - submod / external content]
This serves as the current copy of the roadmap for [Vera](https://github.com/ProjectMonika/Vera), a planned replacement to the current updater for MAS.

### Current working group

- @sr229 (Project Leader, ServOps)
- @wafflemelon (Developer) 

### TODOs:

- [ ] Backwards compatibility with Python 2.7 (Required since RenPy is Py2)
- [ ] Follow "Always-Overwrite" strategy for updates.
- [ ] UI Improvements.
- [ ] Non-blocking updates.

**Comment by ThePotatoGuy** [comment]
> Vera performs a background update and searches for a new version every hour. 

Please change this to once per 24 hours. My wallet does not agree with frequent update checking.

Also, please make this work with the existing file structure Renpy auto-generates when making builds ([link](https://www.renpy.org/doc/html/updater.html)). We're not about to add an extra step in between building an update making it live.
